src/controllers/update/overseas.entity.review.controller.ts

```diff
@@ -31,9 +32,19 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
-export const post = (req: Request, res: Response, next: NextFunction) => {
+export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
-    logger.debugRequest(req, `POST ${config.OVERSEAS_ENTITY_REVIEW_PAGE}`);
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const session = req.session as Session;
```

Comment:
I'm not sure this is the right temporary location. Please move to bo/mo type selection page and only load once.

Reply:
We should already have ensured we have entity_number via navigation controls.

Reply:
Without having the mapping the call only once is tricky but you could check that BO list in Appdata is undefined. Then set it to [] (empty list) instead of mapped list and use that to decide (i.e. not === undefined) if the PSC data needs to be fetched.

Reply:
Thank you, I'll move that to the GET action of the next controller and figure out a solution to only check it once. 
Just one thing to point out, I added the if statement to check for the entity number because lint complains about appData.entity_number as its defined as optional in the ApplicationData model.

Reply:
updated and moved to BO type controller